[2025-10-29T13:28:02.180+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag1_calcul_trajet_kafka.ConsumKafka scheduled__2025-10-29T13:27:00+00:00 [queued]>
[2025-10-29T13:28:02.198+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag1_calcul_trajet_kafka.ConsumKafka scheduled__2025-10-29T13:27:00+00:00 [queued]>
[2025-10-29T13:28:02.199+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-10-29T13:28:02.222+0000] {taskinstance.py:2191} INFO - Executing <Task(_PythonDecoratedOperator): ConsumKafka> on 2025-10-29 13:27:00+00:00
[2025-10-29T13:28:02.238+0000] {standard_task_runner.py:60} INFO - Started process 263 to run task
[2025-10-29T13:28:02.245+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag1_calcul_trajet_kafka', 'ConsumKafka', 'scheduled__2025-10-29T13:27:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/dag1_kafka_processing.py', '--cfg-path', '/tmp/tmpxhrz8yio']
[2025-10-29T13:28:02.271+0000] {standard_task_runner.py:88} INFO - Job 14: Subtask ConsumKafka
[2025-10-29T13:28:02.575+0000] {task_command.py:423} INFO - Running <TaskInstance: dag1_calcul_trajet_kafka.ConsumKafka scheduled__2025-10-29T13:27:00+00:00 [running]> on host c91e5c2c38a7
[2025-10-29T13:28:02.854+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag1_calcul_trajet_kafka' AIRFLOW_CTX_TASK_ID='ConsumKafka' AIRFLOW_CTX_EXECUTION_DATE='2025-10-29T13:27:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-29T13:27:00+00:00'
[2025-10-29T13:28:02.870+0000] {conn.py:396} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <connecting> [IPv4 ('172.18.0.6', 9092)]>: connecting to kafka:9092 [('172.18.0.6', 9092) IPv4]
[2025-10-29T13:28:02.882+0000] {conn.py:618} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <checking_api_versions_recv> [IPv4 ('172.18.0.6', 9092)]>: Broker version identified as 2.6
[2025-10-29T13:28:02.883+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <connected> [IPv4 ('172.18.0.6', 9092)]>: Connection complete.
[2025-10-29T13:28:02.892+0000] {subscription_state.py:181} INFO - Updating subscribed topics to: ('source',)
[2025-10-29T13:28:02.893+0000] {dag1_kafka_processing.py:49} INFO - âœ… Consommateur Kafka crÃ©Ã© pour le topic 'source'. En attente de messages...
[2025-10-29T13:28:02.897+0000] {conn.py:396} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.6', 9092)]>: connecting to kafka:9092 [('172.18.0.6', 9092) IPv4]
[2025-10-29T13:28:02.898+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:9092 <connected> [IPv4 ('172.18.0.6', 9092)]>: Connection complete.
[2025-10-29T13:28:02.899+0000] {conn.py:945} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=bootstrap-0 host=kafka:9092 <connected> [IPv4 ('172.18.0.6', 9092)]>: Closing connection. 
[2025-10-29T13:28:03.004+0000] {cluster.py:401} INFO - Coordinator for group/***-consumers-dag1 is ('coordinator-1', 'kafka', 9092, None)
[2025-10-29T13:28:03.005+0000] {base.py:788} INFO - Discovered coordinator coordinator-1 for group ***-consumers-dag1
[2025-10-29T13:28:03.005+0000] {base.py:848} INFO - Starting new heartbeat thread
[2025-10-29T13:28:03.007+0000] {consumer.py:365} INFO - Revoking previously assigned partitions () for group ***-consumers-dag1
[2025-10-29T13:28:03.008+0000] {base.py:419} INFO - Failed to join group ***-consumers-dag1: NodeNotReadyError: coordinator-1
[2025-10-29T13:28:03.009+0000] {conn.py:396} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=coordinator-1 host=kafka:9092 <connecting> [IPv4 ('172.18.0.6', 9092)]>: connecting to kafka:9092 [('172.18.0.6', 9092) IPv4]
[2025-10-29T13:28:03.010+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=coordinator-1 host=kafka:9092 <connected> [IPv4 ('172.18.0.6', 9092)]>: Connection complete.
[2025-10-29T13:28:03.111+0000] {base.py:532} INFO - (Re-)joining group ***-consumers-dag1
[2025-10-29T13:28:03.114+0000] {base.py:637} INFO - Received member id kafka-python-2.2.15-f06d74e4-6aaa-4dcd-b1f5-08487f9f56b8 for group ***-consumers-dag1; will retry join-group
[2025-10-29T13:28:03.115+0000] {base.py:419} INFO - Failed to join group ***-consumers-dag1: [Error 79] MemberIdRequiredError
[2025-10-29T13:28:03.116+0000] {base.py:532} INFO - (Re-)joining group ***-consumers-dag1
[2025-10-29T13:28:06.122+0000] {base.py:594} INFO - Successfully joined group ***-consumers-dag1 <Generation 6 (member_id: kafka-python-2.2.15-f06d74e4-6aaa-4dcd-b1f5-08487f9f56b8, protocol: range)>
[2025-10-29T13:28:06.124+0000] {base.py:596} INFO - Elected group leader -- performing partition assignments using range
[2025-10-29T13:28:06.128+0000] {subscription_state.py:254} INFO - Updated partition assignment: [('source', 0)]
[2025-10-29T13:28:06.130+0000] {consumer.py:249} INFO - Setting newly assigned partitions (('source', 0),) for group ***-consumers-dag1
[2025-10-29T13:28:06.142+0000] {dag1_kafka_processing.py:60} INFO - ðŸ“© Message reÃ§u : {'confort': 'medium', 'prix_base_per_km': 2, 'properties-client': {'logitude': 2.367099346019275, 'latitude': 48.84588053797688, 'nomclient': 'FALL', 'telephoneClient': '060786575'}, 'properties-driver': {'logitude': 3.723866372098938, 'latitude': 40.38308919751626, 'nomDriver': 'DIOP', 'telephoneDriver': '0760786575'}}
[2025-10-29T13:28:06.145+0000] {base.py:1052} INFO - Stopping heartbeat thread
[2025-10-29T13:28:06.146+0000] {base.py:891} INFO - Leaving consumer group (***-consumers-dag1).
[2025-10-29T13:28:06.150+0000] {base.py:906} INFO - LeaveGroup request for group ***-consumers-dag1 returned successfully
[2025-10-29T13:28:06.151+0000] {conn.py:945} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=1 host=kafka:9092 <connected> [IPv4 ('172.18.0.6', 9092)]>: Closing connection. 
[2025-10-29T13:28:06.151+0000] {conn.py:945} INFO - <BrokerConnection client_id=kafka-python-2.2.15, node_id=coordinator-1 host=kafka:9092 <connected> [IPv4 ('172.18.0.6', 9092)]>: Closing connection. 
[2025-10-29T13:28:06.152+0000] {python.py:201} INFO - Done. Returned value was: {'confort': 'medium', 'prix_base_per_km': 2, 'properties-client': {'logitude': 2.367099346019275, 'latitude': 48.84588053797688, 'nomclient': 'FALL', 'telephoneClient': '060786575'}, 'properties-driver': {'logitude': 3.723866372098938, 'latitude': 40.38308919751626, 'nomDriver': 'DIOP', 'telephoneDriver': '0760786575'}}
[2025-10-29T13:28:06.198+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag1_calcul_trajet_kafka, task_id=ConsumKafka, execution_date=20251029T132700, start_date=20251029T132802, end_date=20251029T132806
[2025-10-29T13:28:06.258+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-10-29T13:28:06.288+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
